### PR TITLE
#972: Stars no longer depends on user/repo parameters. It now has its ow...

### PR DESCRIPTION
**Pull request by pinaf**
[#972](https://github.com/jcabi/jcabi-github/issues/972)

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks for your pull request, let me find someone who can review it

**Comment by dmarkov** [comment]
@carlosmiranda could you please review this pull request

File: src/test/java/com/jcabi/github/RtStarsTest.java [hunk 1]
```diff
@@ -60,14 +61,19 @@ public void checkIfRepoStarred() throws Exception {
             new MkAnswer.Simple(HttpURLConnection.HTTP_NO_CONTENT)
         ).next(new MkAnswer.Simple(HttpURLConnection.HTTP_NOT_FOUND))
             .start();
-        final Stars stars = new RtStars(
-            new ApacheRequest(container.home())
+        final Stars truestars = new RtStars(
```
Comment (carlosmiranda) [inline src/test/java/com/jcabi/github/RtStarsTest.java]: Let's call this variable `stars` or `starred`. Better avoid compound variable names: http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names

File: src/test/java/com/jcabi/github/RtStarsTest.java [hunk 2]
```diff
@@ -60,14 +61,19 @@ public void checkIfRepoStarred() throws Exception {
...
+            new ApacheRequest(container.home()),
+            RtStarsTest.repo("someuser", "starredrepo")
         );
         MatcherAssert.assertThat(
-            stars.starred("someuser", "starredrepo"), Matchers.is(true)
+            truestars.starred(), Matchers.is(true)
+        );
+        final Stars falsestars = new RtStars(
```
Comment (carlosmiranda) [inline src/test/java/com/jcabi/github/RtStarsTest.java]: Let's call this `unstarred` instead. See http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names.

**Comment by carlosmiranda** [comment]
@pinaf A couple of minor comments above.

**Comment by pinaf** [comment]
@carlosmiranda there you go.

**Comment by carlosmiranda** [comment]
@pinaf Thanks, looks good now.

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by dmarkov** [comment]
@carlosmiranda **16 mins** sent to your balance (ID `50342821`), many thanks!

+16 added to your rating, current score is: [+1304](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**